lua/nvim-treesitter/parsers.lua

```diff
@@ -531,7 +531,8 @@ list.faust = {
 list.fennel = {
   install_info = {
     url = "https://github.com/alexmozaidze/tree-sitter-fennel",
-    files = { "src/parser.c" },
+    files = { "src/parser.c", "src/scanner.c" },
+    branch = "dev",
```

Comment:
I don't think we want a prerelease branch here; it's OK to leave the PR up for testing, but whatever gets merged must refer to the default branch.

Reply:
This is a draft. I will either force-push or make a separate PR for when the above TODO list is mostly done, and there are no major bugs.